man/systemd.exec.xml

```diff
@@ -1644,6 +1644,24 @@ StateDirectory=aaa/bbb ccc</programlisting>
         <xi:include href="version-info.xml" xpointer="v234"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>StateDirectoryQuota=</varname></term>
+        <term><varname>CacheDirectoryQuota=</varname></term>
+        <term><varname>LogsDirectoryQuota=</varname></term>
```

Comment:
RuntimeDirectorQuota= too, no? after all tmpfs supports quota on very recent kernels too, no?

Reply:
ah, right, patches have been posted, but not been merged for that. ok, let's leave this out for now.